include/rmm/mr/device/device_memory_resource.hpp

```diff
@@ -18,6 +18,9 @@
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/detail/aligned.hpp>
 
+#include <cuda/memory_resource>
+#include <cuda/stream_ref>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Is this include necessary? `stream_ref` is unused in this file.